internal/cli/global_opts.go

```diff
@@ -109,18 +109,11 @@ func ExampleAtlasEntryPoint() string {
 		return exampleBin
 	}
 	exampleBin = config.BinName()
-	if exampleBin == config.MongoCLI {
```

Comment:
maybe worth dropping the function altogether but that may be a bigger PR, ok for a followup